protos/flyteidl/event/event.proto

```diff
@@ -154,6 +154,14 @@ message ParentNodeExecutionMetadata {
     string node_id = 1;
 }
 
+message BatchedReason {
```

Comment:
nit: seems confusing to call the individual reason message "Batched"

Reply:
+1